tests/entrypoints/openai/basic_tests/test_basic.py

```diff
@@ -11,63 +11,7 @@
 
 from vllm.version import __version__ as VLLM_VERSION
 
-from ...utils import RemoteOpenAIServer
-
-MODEL_NAME = "HuggingFaceH4/zephyr-7b-beta"
-
-
-@pytest.fixture(scope='module')
-def server_args(request: pytest.FixtureRequest) -> list[str]:
-    """ Provide extra arguments to the server via indirect parametrization
-
-    Usage:
-
-    >>> @pytest.mark.parametrize(
-    >>>     "server_args",
-    >>>     [
-    >>>         ["--disable-frontend-multiprocessing"],
-    >>>         [
-    >>>             "--model=NousResearch/Hermes-3-Llama-3.1-70B",
-    >>>             "--enable-auto-tool-choice",
-    >>>         ],
-    >>>     ],
-    >>>     indirect=True,
-    >>> )
-    >>> def test_foo(server, client):
-    >>>     ...
-
-    This will run `test_foo` twice with servers with:
-    - `--disable-frontend-multiprocessing`
-    - `--model=NousResearch/Hermes-3-Llama-3.1-70B --enable-auto-tool-choice`.
-
-    """
-    if not hasattr(request, "param"):
-        return []
-
-    val = request.param
-
-    if isinstance(val, str):
-        return [val]
-
-    return request.param
-
-
-@pytest.fixture(scope="module")
-def server(server_args):
-    args = [
-        # use half precision for speed and memory savings in CI environment
-        "--dtype",
-        "bfloat16",
-        "--max-model-len",
-        "8192",
-        "--enforce-eager",
-        "--max-num-seqs",
-        "128",
-        *server_args,
-    ]
-
-    with RemoteOpenAIServer(MODEL_NAME, args) as remote_server:
-        yield remote_server
+MODEL_NAME = "hmellor/tiny-random-LlamaForCausalLM"
```

Comment:
Could this also be made a fixture so that we don't have to keep anything in sync with conftest?